app/views/admin/media/_index.html.slim

```diff
@@ -0,0 +1,34 @@
+.media-browser
+  - media.each do |media_item|
+    div data-media-id="#{media_item.id}"
+      .col-lg-1.col-md-2.col-sm-3.col-xs-6
+        .thumb
+          = link_to image_tag(media_item.item.thumb.url), media_item.item.url
+          .actions
+            = link_to edit_admin_media_path(media_item, media_attachable_type: owner.class.name.parameterize,  media_attachable_id: owner.id), class: 'edit' do
+              i.fa.fa-pencil
+            = link_to admin_media_path(media_item, media_attachable_type: owner.class.name.parameterize,  media_attachable_id: owner.id), 
+                method: :delete,
+                data: { confirm: t('media.confirm_deletion'),
+                  "confirm-cancel": t(:cancel),
+                  "confirm-proceed": t(:confirm) } do
+              i.fa.fa-trash
+      = render partial: "admin/media/media_form_modal", locals: { media: media_item, action_url: admin_media_path(media_item, media_attachable_id: owner.id, media_attachable_type: owner.class.name) }
```

Comment:
Please split up this line using the \ at end of line. Perhaps a good time to do one-hashkey-per-line?